.github/workflows/build-and-deploy.yml

```diff
@@ -90,7 +90,7 @@ jobs:
               process.env.OWNER,
               process.env.REPO,
               process.env.REF,
-              process.env.BUILD_ONLY ? 'build' : 'deploy',
+              (process.env.BUILD_ONLY ? 'build' : 'deploy') + (process.env.REPO=='MSYS2-packages' && process.env.ARCHITECTURE != '' ? '' : '_' + process.env.ARCHITECTURE),
```

Comment:
I think we also need this for the upcoming ARM64 builds:

```suggestion
              `${
                process.env.BUILD_ONLY ? 'build' : 'deploy'
              }${
                process.env.ARCHITECTURE ? `_${process.env.ARCHITECTURE}` : ''
              }`,
```

Reply:
I think you're right. At first I thought we only needed this for Msys-Packages, because we currently only need one workflow run for Mingw-Packages, but that's about to change.